linuxdoc/kernel_doc.py

```diff
@@ -1035,9 +1032,6 @@ def output_function_decl(
                 # pointer to function
                 p_list.append("%s%s)(%s)"
                               % (self.FUNC_PTR[0], p_name, self.FUNC_PTR[1]))
-            elif p_type.endswith("*"):
-                # pointer
-                p_list.append("%s%s" % (p_type, p_name))
```

Comment:
@nbourdau & @GabrielGanne thanks for pointing out this error. Your solution adds to many spaces. I tried to fix it my way, see commit 4a45417 .. can you please check if this is OK for you?

Reply:
> can you please check if this is OK for you?

I have checked your commit 4a45417 and I would reply: Yes and no :-). It indeed fixes the way the parameters and return type are displayed and this is better than what it was. However:
 - it imposes a coding style which might be different from the documented project
 - it does not normalize everywhere in the type string (for example if you have intermediate * with trailing const, they will not be normalized)

It is fine for the moment, but maybe in future (this is not on my priority list) I will propose commits which does the complete normalization and project option to make this normalization optional.

Reply:
> it imposes a coding style which might be different from the documented project

@nbourdau, good point. I dropped the normalization and reduce the patch to fix only the error reported at the beginning. See commit message: https://github.com/return42/linuxdoc/commit/30cae1e .. can you give it another try?

If normalization is needed, it could be added in the future (I guess a good solution needs more investigation).